### ISSUE TITLE
Upgrade to angular2.beta15?

### ISSUE THREAD

**Issue by BenevidesLecontes** [x6doooo/generator-gulp-angular2#20]
I need to upgrade to beta.15 but i'm facing a lot of errors. Is there a simple way for that?

**Comment by ruffiem** [comment]
Nope, no easy way. There's a whole part of the gulpfile.js to rewrite (where the systemjs-builder is being used). I'm working on it but if you already have a fix, I will take it.

**Comment by ruffiem** [comment]
https://github.com/ruffiem/generator-angular2-application-scaffolder

**Comment by BenevidesLecontes** [comment]
@ruffiem Thanks for your amazing job, I'll try to use it.
